client/src/client_option.rs

```diff
@@ -0,0 +1,18 @@
+use {
+    crate::connection_cache::ConnectionCache,
+    solana_keypair::Keypair,
+    std::{net::UdpSocket, sync::Arc},
+    tokio::runtime::Handle as RuntimeHandle,
+    tokio_util::sync::CancellationToken,
+};
+
+/// [`ClientOption`] enum represents the available client types for TPU
+/// communication:
+/// * [`ConnectionCacheClient`]: Uses a shared [`ConnectionCache`] to manage
+///   connections efficiently.
+/// * [`TpuClientNextClient`]: Relies on the `tpu-client-next` crate and
+///   requires a reference to a [`Keypair`].
+pub enum ClientOption<'a> {
```

Comment:
I wanted to say that it can be used also in forwarding_stage.rs. But the `ForwardingClientOption` now supports multihoming so it is not the same now. to double check, @alexpyattaev am i right that multihoming is irrelevant for rpc and also for the votes over quic?